GliaWidgets/ViewModel/Chat/ChatViewModel.swift

```diff
@@ -362,12 +362,24 @@ extension ChatViewModel {
             })?.offset
         else { return }
 
-        let status = Strings.Message.Status.delivered
-        let message = ChatMessage(with: message)
-        let item = ChatItem(kind: .visitorMessage(message, status: status))
-        downloader.addDownloads(for: message.attachment?.files, with: uploads)
+        let deliveredStatus = Strings.Message.Status.delivered
+
+        // Remove previous "Delivered" statuses
+        section.items
+            .enumerated()
+            .forEach { index, element in
+                if case .visitorMessage(let message, let status) = element.kind,
```

Comment:
`if case let .visitorMessage(message, status) = element.kind` perhaps?

Reply:
AFAIK there's no difference between the two. Your suggestion is a bit shorter, but I've just been following the code style of the project.

Reply:
I was just suggesting since the shorter length, yep. If thats the general code style then all good.